addons/scope_remover/fn_init.sqf

```diff
@@ -9,12 +9,12 @@
     true   /* needRestart */
 ] call CBA_settings_fnc_init;
 
-if !(isServer) exitWith {};
 
 ["CBA_settingsInitialized", {
     if !(cnto_scope_remover_enable) exitWith {};
     ["CAManBase", "initPost", {
         params ["_unit"];
+        if !(local _unit) exitWith {};
         if (primaryWeapon _unit == "" || _unit in playableUnits || isPlayer _unit || _unit in allPlayers) exitWith {};
```

Comment:
Aren't last three conditions overlapping?

Reply:
Yes, just want to be _really_ sure we're not messing with players.